evmbin/src/main.rs

```diff
@@ -68,8 +68,8 @@ use bytes::Bytes;
 use ethcore::{spec, json_tests, TrieSpec};
 use vm::{ActionParams, CallType};
 
-mod info;
-mod display;
+pub mod info;
```

Comment:
Is it actually needed for this to be `pub`?

Reply:
I had to make it `pub` otherwise it doesn't show you any of the documentation contained in that `info` module when you build and open the the docs for the evmbin package with `cargo doc -p evmbin --release --open --no-deps`

Reply:
Hmm. It's a shame if we have to expose a larger API surface just so we can show docs... If `info` is not public then we probably should not make it so just to document it. Maybe [this](https://stackoverflow.com/questions/29378113/how-can-i-include-private-modules-when-generating-documentation-via-cargo) is useful here?

Reply:
Thanks, yes that link worked great. I've pushed changes in this commit 41b2285b7 that reverts the changes so those modules aren't public again. Their docs may be viewed with `cargo doc -p evmbin --release --open --no-deps --document-private-items`